### DISCUSSION TITLE
12v Battery Issues 2023 Kia Sorento

### DISCUSSION THREAD

**Discussion by sbcrumb** [Hyundai-Kia-Connect/kia_uvo#847] [General]
Since I got my car I have been chasing 12V issues getting your car is below 40% in the kia app after time.
I have had it at the dealer since Jan 2nd 2024, today a field rep from KIA came out and said the drain was coming from the Smart Phone App(which is there app) or a different app that has permission and is pinging the car and waking up modules. I have the HA integration set at 1440 so once a day. Yet the battery trend will show it waiting up more/differently then every 24 hours, cause when it does wake up after 24 hours the drain can drop 1% to 3%.

And for the fun part, my wife has a 2023 Kia Sportage app on her phone, and same settings in HA. And Zero issue with hers. The field tech couldn't explain that. But still left with saying it is happening to my car cause of the app or "Something Else"

Any one have any suggestions to prove them wrong? Or prove it is there side doing the wake up?

**Comment by cdnninja** [comment]
Turn on debug mode and watch to see if a force refresh is sent often or not.  If not you can confirm it is following the 1440 as intended.    Other option is naturally disabling this integration and seeing if you still have issues.

**Comment by MobilettoSoft** [comment]
On my Tucson PHEV I had a  warning on the cockpit saying that the 12v battery was discharging due to external devices for about the last 6 months, every morning. Battery levels on HA were in the 39-60% ballpark. Service said nothing was wrong and that it was an expected behavior. After wandering online for a solution, I tried disconnecting the 12v battery for about 20 minutes. Afterward, the warning on the cockpit disappeared and the 12v battery levels in HA jumped to 70 to 95%. I don't know if it was just a coincidence, but somehow it seemed to work (maybe resetting some hardware or software components).

**Comment by Joseph9052** [comment]
Hi sbcrumb, i have question the problem is solved ?

**Comment by sbcrumb** [comment]
Yes KIA bought the car back. Lemon law
